### ISSUE TITLE
Standalone version

### ISSUE THREAD

**Issue by meecect** [jagi/meteor-astronomy#727]
Have you ever thought or had any plans of producing a stand-alone version of this?  I'm not sure how tightly coupled to Meteor this is, but it strikes me as a very good and easy to use ORM for node and mongo-based apps.

thanks,
Cliff

**Comment by lukejagodzinski** [comment]
Yep at some point I though about it. However, now I changed my mind and honestly I think ORMs are only good for very simple CRUD operations and for everything else you can write queries manually. It's especially easy with MongoDB. I'm sold on the idea of code generators, especially for SQL and GraphQL.
